webrtc.html

```diff
@@ -6675,8 +6789,9 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
 
       <tbody>
         <tr>
-          <td><code id=
-          "event-dtlstransport-statechange">statechange</code></td>
+          <td><dfn id="event-dtlstransport-statechange"
+                   data-lt="RTCDtlsTransport state change">
```

Comment:
please add a `data-lt-noDefault` attribute to avoid the duplicate anchor bug in respec